### DISCUSSION TITLE
--target filesystem in repro or  github artifacts?

### DISCUSSION THREAD

**Discussion by ghost** [GoogleChrome/lighthouse-ci#515] [Q&A]
Hello,

I try to get the files directly in the repro.
The following works without any issues:
      - run: lhci upload --target temporary-public-storage

Now I am trying to output it directly in my repro with:
      - run: lhci upload --target filesystem --outputDir ./reports
      
That works but I have no access to the files. Do I need to use github artifacts? 

Many thanks,
Sven

**Comment by patrickhulce** [comment]
I'm not sure I understand what you're trying to accomplish. Are you using GitHub Actions and you want to see the report in there instead of in public storage?

> Now I am trying to output it directly in my repro with:
> - run: lhci upload --target filesystem --outputDir ./reports

This will put the reports into a local directory called "reports". Is that what you're trying to reproduce? Or did you mean **repo** and you're trying to add the reports to your repository?
